CONTRIBUTING.md

```diff
@@ -1 +1,81 @@
+# Introduction
+
+### You want to help out with Aether? Yay!
+
+First off, thank you for considering contributing to Aether. It's people like you that make Aether *the* platform for data-driven humanitarian projects.
+
+## How you can help
+
+Aether is an open source project and we love to receive contributions from our community. There are many ways to contribute, from writing tutorials or blog posts, improving the documentation, submitting bug reports and feature requests or writing code which can be incorporated into Aether itself. 
+
+If you decide that you want to help out with development, you should start by getting Aether installed locally. Follow the [guide](https://aether.ehealthafrica.org/documentation/try/index.html) on the Aether website (and of course, if you have any problems following that guide, or you think it could be improved in any way, feel free to open and issue!). Once you’ve got Aether running and you’ve got a feel for the fundamental concepts and architecture, then you can dive right in and start writing code.
+
+At any point, you can get help on the [Aether forum](https://forums.ehealthafrica.org/c/aether).
+
+Our [Jira board](https://jira.ehealthafrica.org/secure/RapidBoard.jspa?rapidView=161&view=planning.nodetail&epics=visible) shows a list of the bugs and features that are currently in our roadmap; you might want to look through them to get a sense of where we’re heading in the near future. You might even want to pick one of them and get to work; if so, fantastic! Get in touch with us on the forum and we can help you figure out the best way to approach the work.
+
+If you find a bug that you want to report, or a feature that you think Aether needs, start by opening and issue on the repo. We can then check it, discuss it with you and ensure that you have added all the required information. Once that’s all done, we will promote the issue to the Jira backlog – at which point you are of course very welcome to start working on it yourself!
+
+## How to report a bug
+
+If you find a security vulnerability, do NOT open an issue. Email aether@ehealthafrica.org instead.
+
+In order to determine whether you are dealing with a security issue, ask yourself these two questions:
+- Can I access something that's not mine, or something I shouldn't have access to?
+- Can I disable something for other people?
+
+If the answer to either of those two questions are "yes", then you're probably dealing with a security issue. Note that even if you answer "no" to both questions, you may still be dealing with a security issue, so if you're unsure, just email us.
+
+When filing an issue, make sure to answer these five questions:
+>
+1. What version of Aether are you using?
+2. What operating system and processor architecture are you using?
+3. What did you do?
+4. What did you expect to see?
+5. What did you see instead?
```

Comment:
Maybe for those question qe want to include a `TEMPLATE_ISSUE.md` file.

Check these ones as an example:
https://github.com/facebook/react/tree/master/.github